target/linux/ramips/image/mt7621.mk

```diff
@@ -203,6 +208,23 @@ define Device/dlink_dir-860l-b1
 endef
 TARGET_DEVICES += dlink_dir-860l-b1
 
+define Device/dlink_dir-878-a1
+  SOC := mt7621
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 16000k
+  DEVICE_VENDOR := D-Link
+  IMAGES += factory.bin
+  DEVICE_MODEL := DIR-878
+  DEVICE_VARIANT := A1
+  DEVICE_PACKAGES := kmod-mt7615e wpad-basic
+  KERNEL_INITRAMFS := $$(KERNEL) | ubootpad96
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | ubootpad96 |\
+	pad-rootfs |  check-size $$$$(IMAGE_SIZE) | append-metadata
+  IMAGE/factory.bin := append-kernel | append-rootfs | ubootpad96 |\
+  	check-size $$$$(IMAGE_SIZE)
```

Comment:
There's a final formatting issue here:

```
  IMAGE/factory.bin := append-kernel | append-rootfs | ubootpad96 |\
  	check-size $$$$(IMAGE_SIZE)
^^
```
Other than that the whitespace issues are handled indeed. About to flash and report back :smile:

Reply:
not sure on this i may have fix something before you posted
the Image tags are double space in the rest of the file not Indent ?